src/rules/data-quality/activity-in-activity-list-rule.js

```diff
@@ -92,7 +92,7 @@ module.exports = class ActivityInActivityListRule extends Rule {
     };
   }
 
-  validateField(node, field) {
+  async validateFieldAsync(node, field) {
```

Comment:
This function is the same as `validateField` except the one line where it uses `await JsonLoaderHelper.getFileAsync`